### DISCUSSION TITLE
What type of components the community wants?

### DISCUSSION THREAD

**Discussion by tiaguinho** [tiaguinho/material-community-components#191] [Ideas]
If you want a component that this Material Community Component doesn't have, this is the right place to ask/vote for new components to be created.

If you see the component in the comments, please vote with a "like" emoji.
If you don't see, just add a comment describing your idea.

**Comment by Splaktar** [comment]
For the Angular Material community, you can see the top voted feature requests [here](https://github.com/angular/components/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc). The number 1 request is the time picker, which you already have. I would suggest looking at the other top requests to see if any of them interest you. Then I would check to see if there is a recent response from the team indicating whether they can work on it soon or not.

**Comment by LukasN** [comment]
It would be great to have a component, that combines date picker of material and a timepicker.  the current workaround for us is to mix your timepicker & material date picker with two trigger buttons. A date time picker that wraps both would be a great help.

**Comment by Splaktar** [comment]
There has been a lot of interest in a fab-speed-dial (https://github.com/angular/components/issues/860). There are a number of third party implementations in StackBlitz and NPM to use as inspiration. It might be a good fit for this repo as the Components team has no plans to implement it at this time.

**Comment by chriszrc** [comment]
Popover! It's fairly high on the angular material wishlist, and is such a common component, I find it hard to believe it hasn't been implemented. As the issue notes, it's similar to a mat-menu, except you could position it anywhere around an element you want, and it could contain what content you give it:

https://github.com/angular/components/issues/2691

**Comment by ayeletdn** [comment]
Range slider. The current slider component has a min/max option to prevent getting out of bounds, but if the user wants to set their own boundaries, there's not simple visual component for this.

https://github.com/angular/components/issues/1331

**Comment by Splaktar** [comment]
> There has been a lot of interest in a fab-speed-dial ([angular/components#860](https://github.com/angular/components/issues/860)). There are a number of third party implementations in StackBlitz and NPM to use as inspiration. It might be a good fit for this repo as the Components team has no plans to implement it at this time.

This is now available here: https://tiaguinho.github.io/material-community-components/speed-dial

**Comment by Splaktar** [comment]
It would be nice to have these
- Bottom Nav like requested in https://github.com/angular/components/issues/408
- Top App Bar as requested in https://github.com/angular/components/issues/13964
- Bottom App Bar as requested in https://github.com/angular/components/issues/16367

Bottom Nav and Bottom App Bar are probably needed the most since MDC has [Top App Bars](https://material.io/develop/web/components/top-app-bar) ([demo](https://material-components.github.io/material-components-web-catalog/#/component/top-app-bar)) now.

**Comment by ghost** [comment]
Bottom Drawer https://material.io/components/navigation-drawer#bottom-drawer

![image](https://user-images.githubusercontent.com/67913260/91912467-30b00600-ec81-11ea-9054-469ae797714f.png)
